definitions/tools/star_align.cwl

```diff
@@ -0,0 +1,190 @@
+        inputBinding:
+            position: 6
+            prefix: '--genomeDir'
+        doc: |
+          string: paths to the directory where genome files are stored, created using genomeGenerate
+    outSAMunmapped:
+        type: int?
+        inputBinding:
+            position: 7
+            prefix: '--outSAMunmapped'
+        doc: |
+          string: output of unmapped reads in the SAM format
+          None   ... no output
+          Within ... output unmapped reads within the main SAM file (i.e. Aligned.out.sam)
+    alignIntronMin:
+        type: int?
+        inputBinding:
+            position: 8
+            prefix: '--alignIntronMin'
+        doc: 'minimum intron size: genomic gap is considered intron if its length>=alignIntronMin,otherwise it is considered Deletion
+        '
+    alignIntronMax:
+        type: int?
+        inputBinding:
+            position: 9
+            prefix: '--alignIntronMax'
+        doc: 'maximum intron size: genomic gap is considered intron if its length>=alignIntronMin,otherwise it is considered Deletion
+        '
+    alignMatesGapMax:
+        type: int?
+        inputBinding:
+            position: 10
+            prefix: '--alignMatesGapMax'
+        doc: 'maximum gap between two mates, if 0, max intron gap will be determined by (2^winBinNbits)*winAnchorDistNbins
+        '
+    outSAMstrandField:
+        type: string
+        default: intronMotif
+        inputBinding:
+            position: 11
+            prefix: '--outSAMstrandField'
+        doc: |
+          None
+          string: Cufflinks-like strand field flag
+          None        ... not used
+          intronMotif ... strand derived from the intron motif. Reads with inconsistent and/or non-canonical introns are filtered out.
+          NOTE you need this option to be set yto intronmotif if for stringtie quantification
+    twopassMode:
+        type: string?
+        inputBinding:
+            position: 12
+            prefix: '--twopassMode'
+        doc: |
+           None
+           string: 2-pass mapping mode.
+           None        ... 1-pass mapping
+           Basic ... basic 2-pass mapping, with all 1st pass junctions inserted into the genome indices on the fly
+    outFilterMultimapNmax:
+        type: int?
+        inputBinding:
+            position: 13
+            prefix: '--outFilterMultimapNmax'
+        doc: 'int: read alignments will be output only if the read maps fewer than this value,otherwise no alignments will be output
+        '
+    sjdbOverhang:
+        type: string?
+        inputBinding:
+            position: 14
+            prefix: '--sjdbOverhang'
+        doc: 'int>0: length of the donor/acceptor sequence on each side of the junctions, ideally = (mate_length - 1)
+        '
+    gtf_file:
+        type: File?
+        inputBinding:
+            position: 15
+            prefix: '--sjdbGTFfile'
+        doc: |
+          string: path to the GTF file with annotations
+    outFileNamePrefix:
```

Comment:
this not a default for star because you would have supplied the GTF during index creation step. 
However for Stringtie step this is default. But I see the point just to keep things consistent ill make this default.